pallets/asset-manager/src/lib.rs

```
@@ -564,7 +564,7 @@ pub mod pallet {
 					Self::get_para_id_from_multilocation(location.into().as_ref())
 				{
 					if para_id != 2084 {
-						Self::increase_count_of_associated_assets(para_id);
+						let _ = Self::increase_count_of_associated_assets(para_id);
```

the call can fail in line 512. That error will be discarded. Why is that safe?
This behavior was the same before, but now it's explicitly tolerated and I don't see why.

Shouldn't we abort migration if the storage item can not be migrated cleanly?

---

I don't think there is going back for migration. (cc @Dengjianping )

---

correct, but we can test with `try_runtime` before deploying to the chain.

This test would currently pass